src/Symfony/Bundle/SecurityBundle/CHANGELOG.md

```diff
@@ -4,6 +4,67 @@ CHANGELOG
 8.0
 ---
 
+ * [BC BREAK] Remove the deprecated `hide_user_not_found` configuration option, use `expose_security_errors` instead
+   
+   Before:
```

Comment:
Too detailed for Symfony's Changelog. But nice one, though.

Reply:
I think this is needed but should be in the upgrade Md file